it/missione.md

```diff
@@ -16,15 +16,35 @@ Il “sistema operativo” del Paese: una serie di componenti fondamentali sui q
 ## Missione
 
 Rendere i servizi pubblici per i cittadini e aziende accessibili nel modo più semplice possibile, innanzitutto tramite dispositivi mobili (approccio “mobile first”), con architetture sicure, scalabili, altamente affidabili e basate su interfacce applicative (API) chiaramente definite;  supportare le pubbliche amministrazioni centrali e locali nel prendere decisioni migliori e il più possibile basate sui dati, grazie all'adozione delle più moderne metodologie di analisi e sintesi dei dati su larga scala, quali Big Data e Machine Learning.
-
-## Attività
-
+In particolare vogliamo:
 1. Coordinare i diversi “stakeholder” pubblici nella gestione di programmi digitali, esistenti e futuri, in maniera integrata con una metodologia agile e un approccio open data
 2. Individuare nuove iniziative di trasformazione digitale e tecnologica
 3. Diventare un centro autorevole di competenza digitale e innovazione per gli “stakeholder” pubblici, allo scopo di condividere linee guida, direttive e pareri
 4. Creare una comunità internazionale di sviluppatori e designer che contribuiscano allo sviluppo delle API e dei servizi digitali, che identifichino e risolvano problemi tecnologici, forniscano informazioni e istruzioni sui principi di innovazione digitale e creino strumenti e servizi condivisi
 5. Porre le fondamenta per un'architettura in evoluzione in grado di "scalare" nel tempo e rimanere al passo con le tendenze tecnologiche che emergeranno
 
+I progetti del team:
+
+[Anagrafe Nazionale Popolazione Residente (ANPR)](https://teamdigitale.governo.it/it/projects/anpr.htm)
```

Comment:
meglio usare link relativi qui, cosi' funzionano anche in locale

Reply:
Lascerei questa regola solo quando facciamo siti nuovi, sii gentile @cloudify non posso riscrivere i link se non è indispensabile. La vita è breve